PEA/Makefile

```diff
@@ -15,6 +18,15 @@ set_jdk:
 
 all: set_jdk
 	javac *.java
+	cd ${CTW_DIR}; make
 
 run: set_jdk
 	./auto.sh
+
+run-ctw: set_jdk
+	cd ${CTW_DIR}/dist; \
```

Comment:
Should we use pushd/popd here? that will automatically resume the working directory.

Reply:
I think it does not matter here. My understanding for Make is that bash commands are run in a subshell. After we `cd`, we are back to the original working directory.